incubator/cloudbeaver/templates/statefulsets.yaml

```diff
@@ -0,0 +1,159 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "cloudbeaver.fullname" . }}
+spec:
+  serviceName: cloudbeaver
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cloudbeaver
+  template:
```

Comment:
Should we also add annotation with config files checksum to trigger rolling upgrade on configuration changes?

Reply:
I thought about doing this, but the thing is that Cloudbeaver only uses these configs on first start.  Later changes won't have any effect.  

That said... I did in the end add the option to _not_ persist configuration, in which case this could be useful.  I'll also add commentary to that effect.

Reply:
Done!